### PR TITLE
doc: Allow forwarded traffic to the lxd bridge for proxy nat devices 

### PR DESCRIPTION
**Pull request by johanehnberg**
Signed-off-by: Johan Ehnberg <johan@molnix.com>

### CODE REVIEW

File: doc/howto/network_bridge_firewalld.md [hunk 1]
```diff
@@ -73,17 +73,19 @@ Depending on your use case, you might need more advanced rules and the example c
 ### UFW: Add rules for the bridge
 
 If UFW has a rule to drop all unrecognized traffic, it blocks the traffic to and from the LXD bridge.
-In this case, you must add rules to allow traffic to and from the bridge.
+In this case, you must add rules to allow traffic to and from the bridge, as well allow traffic as forwarded to it.
```
Comment (tomponline) [inline doc/howto/network_bridge_firewalld.md:76]: "as well as allowing traffic that is forwarded to it" reads better in my opinion.

Comment (johanehnberg) [reply doc/howto/network_bridge_firewalld.md]: I agree, fixed

**Review by tomponline — APPROVED** [review]
LGTM, thanks!

**Review by tomponline — CHANGES_REQUESTED** [review]
Please can you rebase your PR into a single signed off commit.

**Comment by johanehnberg** [comment]
Sorry, seems I cannot get rid of the second commit with either squash or fixup. The git docs are no t helping me here.

In GitHub, an MR can simply be marked for squashing resulting in a single commit.

**Comment by tomponline** [comment]
I use:

```
git rebase -i master
```

Then find the commit I want to squash into the one above and change `pick` to `f` (for fixup).

**Comment by johanehnberg** [comment]
Good. Going through master worked.

**Pull request merged**